src/geom/elem.C

```diff
@@ -399,7 +400,7 @@ dof_id_type Elem::key () const
 {
   const unsigned short n_n = this->n_nodes();
 
-  std::vector<dof_id_type> node_ids(n_n);
+  std::array<dof_id_type, Elem::max_n_nodes> node_ids;
```

Comment:
Wait, we are now sorting (and hashing) an array with 27 entries every time regardless of the number of nodes the Elem has, and this is faster?

Reply:
No, this is me forgetting to rejigger the end iterator of the sort call.